packages/loader/container-loader/src/retriableDocumentStorageService.ts

```diff
@@ -46,7 +46,9 @@ export class RetriableDocumentStorageService implements IDocumentStorageService,
         return runWithRetry(
             async () => this.internalStorageService.getSnapshotTree(version),
             "storage_getSnapshotTree",
```

Comment:
Consider creating RetriableDocumentStorageService.runWithRetry(Core) that accepts only two args and fills in the rest when calling runWithRetry